frontend/internal-packages/agent/src/chat/workflow/types.ts

```diff
@@ -11,6 +11,8 @@ export type WorkflowState = {
   projectId?: string | undefined
   error?: string | undefined
 
+  ddlStatements?: string | undefined
```

Comment:
(aside)(don't need to fix) I thought it would probably be `string[]` since DDL could be created multiple times during the trial and error process.